systemtest/playlist.xml

```diff
@@ -0,0 +1,340 @@
+		<command>export JAVA_HOME=$(JDK_HOME)$(D)  && \
+	perl $(TEST_RESROOT)$(D)stf$(D)stf.core$(D)scripts$(D)stf.pl  \
+	-test-root=$(Q)$(TEST_RESROOT)$(D)stf;$(TEST_RESROOT)$(D)openjdk-systemtest$(Q) \
+	-systemtest-prereqs=$(Q)$(TEST_RESROOT)$(D)systemtest_prereqs$(Q)  \
+	-results-root=$(REPORTDIR)  \
+	-test=NioLoadTest; \
+	$(TEST_STATUS)</command>	
+		<tags>
+			<tag>extended</tag>
+		</tags>
+		<subsets>
+			<subset>SE80</subset>
+			<subset>SE90</subset>
+		</subsets>
+	</test>
+	
+	<test>
+		<testCaseName>UtilLoadTest</testCaseName>
+		<variations>
+			<variation>NoOptions</variation>
+		</variations>
+		<command>export JAVA_HOME=$(JDK_HOME)$(D)  && \
+	perl $(TEST_RESROOT)$(D)stf$(D)stf.core$(D)scripts$(D)stf.pl  \
+	-test-root=$(Q)$(TEST_RESROOT)$(D)stf;$(TEST_RESROOT)$(D)openjdk-systemtest$(Q) \
+	-systemtest-prereqs=$(Q)$(TEST_RESROOT)$(D)systemtest_prereqs$(Q)  \
+	-results-root=$(REPORTDIR)  \
+	-test=UtilLoadTest; \
+	$(TEST_STATUS)</command>	
+		<tags>
+			<tag>extended</tag>
+		</tags>
+		<subsets>
+			<subset>SE80</subset>
+			<subset>SE90</subset>
+		</subsets>
+	</test>
+	
+	<test>
+		<testCaseName>LambdaLoadTest</testCaseName>
+		<variations>
+			<variation>NoOptions</variation>
+		</variations>
+		<command>export JAVA_HOME=$(JDK_HOME)$(D)  && \
+	perl $(TEST_RESROOT)$(D)stf$(D)stf.core$(D)scripts$(D)stf.pl  \
+	-test-root=$(Q)$(TEST_RESROOT)$(D)stf;$(TEST_RESROOT)$(D)openjdk-systemtest$(Q) \
+	-systemtest-prereqs=$(Q)$(TEST_RESROOT)$(D)systemtest_prereqs$(Q)  \
+	-results-root=$(REPORTDIR)  \
+	-test=LambdaLoadTest; \
+	$(TEST_STATUS)</command>	
+		<tags>
+			<tag>extended</tag>
+		</tags>
+		<subsets>
+			<subset>SE80</subset>
+			<subset>SE90</subset>
+		</subsets>
+	</test>
+	
+	<test>
+		<testCaseName>JdiTest</testCaseName>
+		<variations>
+			<variation>NoOptions</variation>
+		</variations>
+		<command>export JAVA_HOME=$(JDK_HOME)$(D)  && \
+	perl $(TEST_RESROOT)$(D)stf$(D)stf.core$(D)scripts$(D)stf.pl  \
+	-test-root=$(Q)$(TEST_RESROOT)$(D)stf;$(TEST_RESROOT)$(D)openjdk-systemtest$(Q) \
+	-systemtest-prereqs=$(Q)$(TEST_RESROOT)$(D)systemtest_prereqs$(Q)  \
+	-results-root=$(REPORTDIR)  \
+	-test=JdiTest; \
+	$(TEST_STATUS)</command>	
+		<tags>
+			<tag>extended</tag>
+		</tags>
+		<subsets>
+			<subset>SE80</subset>
+			<subset>SE90</subset>
+		</subsets>
+	</test>
+	
+	<test>
+		<testCaseName>HCRLateAttachWorkload</testCaseName>
+		<variations>
+			<variation>NoOptions</variation>
+		</variations>
+		<command>export JAVA_HOME=$(JDK_HOME)$(D)  && \
+	perl $(TEST_RESROOT)$(D)stf$(D)stf.core$(D)scripts$(D)stf.pl  \
+	-test-root=$(Q)$(TEST_RESROOT)$(D)stf;$(TEST_RESROOT)$(D)openjdk-systemtest$(Q) \
+	-systemtest-prereqs=$(Q)$(TEST_RESROOT)$(D)systemtest_prereqs$(Q)  \
+	-results-root=$(REPORTDIR)  \
+	-test=HCRLateAttachWorkload; \
+	$(TEST_STATUS)</command>	
+		<tags>
+			<tag>extended</tag>
+		</tags>
+		<subsets>
+			<subset>SE80</subset>
+			<subset>SE90</subset>
+		</subsets>
+	</test>
+	
+</playlist>

```

Comment:
As per our earlier discussion, thank you for stripping out the extra make layer provided by systemtest, as I think keeping the layers of indirection to a minimum will help us going forward to evolve our tools to the leanest, most necessary set possible (and easier to debug and maintain).

Reply:
Also, as an initial playlist this is good.  We can add some additional tags, to be even more flexible on how we categorize these tests, once we move to using the upgraded testkitgen tooling.

Reply:
Be aware that in "stripping out the extra make layer" you lose the provided capabilities to generate correctly formatted Windows vs unix command lines (slashes, quotes etc.).

Reply:
We have $Q and $D already, so its a good point.  @Mesbah-Alam please update the playlist.xml file to use $Q for quotes and $D for delimiter (slash) - look at the openjdk_regression/playlist for examples.  

To answer the other question, if I can strip tooling to its simplest form, there is a hope of it not being a massive mess and maintenance nightmare in the future.  That is what we choose to do when we fetch and run openjdk regression tests, and that is what we choose to do when we grab the system tests.  

We are already pulling in an extra (STF) framework for running the JCKs that is not actually necessary to actually run them (as we have done in the past), but possibly offers some convenience.  I won't go so far as suggesting we also strip that layer out in the upcoming JCK PR, but when I look ahead a year or two from now, my motivation is to make sure we make choices that simplify and reduce, rather than add and make more complex.  

The detritus of tooling accumulates quickly and becomes unmanageable over time, my mission is to avoid the accumulation when possible.

Reply:
@smlambert , Hi Shelley, I am using $Q for quotes and $D for delimiter (slash) in the playlist.xml.

Reply:
Has the playlist been tested on Windows?